xds/src/main/java/io/grpc/xds/GcpAuthenticationFilter.java

```diff
@@ -219,4 +221,32 @@ V getOrInsert(K key, Function<K, V> create) {
       return cache.computeIfAbsent(key, create);
     }
   }
+
+  /**
+   * Parser for Audience metadata type.
+   */
+  static class AudienceMetadataParser implements ClusterMetadataValueParser {
+
+    @Override
+    public String getTypeUrl() {
+      return "type.googleapis.com/envoy.extensions.filters.http.gcp_authn.v3.Audience";
```

Comment:
Hmm... at some point we should probably swap away from this pattern (used mulitple places) to `"type.googleapis.com/" + Audience.getDefaultInstance().getDescriptorForType().getFullName()` along with a helper. This is fine for now.